src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1000,6 +1000,7 @@
     <Field Name="LocalSymbol" Type="LocalSymbol"/>
     <Field Name="IsDeclaration" Type="bool"/>
     <Field Name="ConstantValueOpt" Type="ConstantValue" Null="allow"/>
+    <Field Name="IsNullableUnknown" Type="bool"/>
```

Comment:
Consider adding a comment

Reply:
Consider flipping to a positive `IsNullableKnown`

Reply:
Added a comment. I've left the field as `IsNullableUnknown` because the common case is the default value `false`, although in general, positive `bool` fields are preferred.